llvm/docs/AMDGPUUsage.rst

```diff
@@ -1438,7 +1438,10 @@ The AMDGPU backend supports the following LLVM IR attributes.
                                              :ref:`amdgpu_ps <amdgpu_ps>` shaders. Any bits enabled by this value will
                                              be enabled in the final register value.
 
-    "amdgpu-wave-priority-threshold"         ..TODO:: Describe.
+    "amdgpu-wave-priority-threshold"         VALU instruction count threshold for adjusting wave priority. If exceeded,
```

Comment:
ping @kosarev

Reply:
The description looks correct to me. The pass was never proven useful though, if I recall correctly.